target/linux/imx6/image/lumi.mk

```diff
@@ -0,0 +1,33 @@
+#
+
+define Device/xiaomi_gateway-lumi
+  DEVICE_VENDOR := Xiaomi
+  DEVICE_MODEL := Lumi Gateway
+  BOARD_NAME := xiaomi,gateway-lumi
+  DEVICE_PACKAGES := kmod-bluetooth \
+  	kmod-sound-core kmod-button-hotplug kmod-bluetooth \
+  	bluez-daemon bluez-libs bluez-utils alsa-utils hostapd \
+  	wpa-supplicant ca-certificates kmod-gpio-button-hotplug \
+  	kmod-i2c-core kmod-iio-core nand-utils
+
+  KERNEL_INSTALL := 0
+  KERNEL_NAME := zImage
+  KERNEL_SUFFIX := -zImage
+  KERNEL := kernel-bin
+  IMAGES := sysupgrade.bin dtb rootfs.bin
+  IMAGE/sysupgrade.bin := sysupgrade-tar | append-metadata
+  IMAGE/rootfs.bin := append-rootfs
+  IMAGE/dtb := install-dtb
+
+  DEVICE_DTS := \
+	imx6ull-xiaomi-lumi
+endef
+
+
+define Device/xiaomi_dgnwg05lm
+  $(Device/xiaomi_gateway-lumi)
+  DEVICE_MODEL := DGNWG05LM (RTL8723BS wifi)
+  DEVICE_PACKAGES += kmod-rtl8723bs rtl8723bs-firmware
+  SUPPORTED_DEVICES += xiaomi,gateway-lumi
```

Comment:
Why add a second name here?

Reply:
The default supported device is "xiaomi,dgnwg05lm" which doesn't match the one in the DTB compatible section.
I want to use the same DTB for both zigbee gateways and allow sysupgrade at the same time.

I use "xiaomi,gateway-lumi" for both devices in dtb compatible.

Reply:
The compatible should be unique. Please create different DTS files with different compatible for different devices.

Apart from that, note the difference between "+=" and ":=".